library/core/src/slice/iter.rs

```diff
@@ -107,24 +107,20 @@ impl<'a, T> Iter<'a, T> {
 
     /// Views the underlying data as a subslice of the original data.
     ///
-    /// This has the same lifetime as the original slice, and so the
```

Comment:
Why remove this?

Reply:
AFAICT this is just normal borrowing rules and does not seem especially relevant here.